app/views/resources/show.html.haml

```diff
@@ -0,0 +1,5 @@
+.row#main_container
+  #left_nav_index{ class: 'small-2 columns' }= render 'students/left_nav_index'
+  #resource_show_conatiner{ class: 'small-10 columns' }
+    %h1= "#{@resource.title}"
```

Comment:
`= "..."` should be rewritten as `...`<br>`%... \#{expression}` can be written without interpolation as `%...= expression`